setup.py

```diff
@@ -9,10 +9,10 @@
     "attrs",
     "orjson",
     "pydantic",
-    "stac_pydantic==3.0.*",
-    "stac-fastapi.api~=3.0.0a0",
-    "stac-fastapi.extensions~=3.0.0a0",
-    "stac-fastapi.types~=3.0.0a0",
+    "stac_pydantic==3.1.*",
```

Comment:
I wonder if we could just remove this, because it's already defined in stac-fastapi.types

Reply:
We could but is it good to show Users what version of stac_pydantic is being used? It's probably better to remove it - cleaner. I don't know.